lua/entities/gmod_wire_expression2/core/globalvars.lua

```diff
@@ -30,24 +30,10 @@ registerType( "gtable", "xgt", {},
 
 __e2setcost(1)
 
-registerOperator("ass", "xgt", "xgt", function(self, args)
-	local lhs, op2, scope = args[2], args[3], args[4]
-	local      rhs = op2[1](self, op2)
-
-	local Scope = self.Scopes[scope]
-	local lookup = Scope.lookup
-	if !lookup then lookup = {} Scope.lookup = lookup end
-	if lookup[rhs] then lookup[rhs][lhs] = true else lookup[rhs] = {[lhs] = true} end
-
-	Scope[lhs] = rhs
-	Scope.vclk[lhs] = true
-	return rhs
+registerOperator("is", "xgt", "n", function(state, this)
```

Comment:
why is this no longer using e2function syntax?

Reply:
Changed this before I rewrote the extpp. It's entirely just styling if you prefer the editor breaking extpp or this for the `is` operator at least.

Reply:
it just seemed random. should be reverted since it doesn't need to change.

Reply:
addressed in summary post => resolving